Editorial review: Document Wasm bulk memory operations (#44296)
mdn/content: files/en-us/webassembly/reference/elem/index.md

---
title: WebAssembly elem instructions
slug: WebAssembly/Reference/Elem
page-type: landing-page
sidebar: webassemblysidebar
---

This set of pages details the instructions available in Wasm related to [elem](/en-US/docs/WebAssembly/Reference/Definitions/elem) definitions.

> - [`elem.drop`](/en-US/docs/WebAssembly/Reference/Elem/drop)

Reviewer: As noted for `data.drop`, this instruction should be cross referenced in the appropriate pages if code is expected to drop `elem` once it is assigned to a table.

Author: In the same way as for `data`, this seems to make the most sense added to the passive `elem` description on the `elem` definition page. I've therefore given this the same treatment.

---
title: WebAssembly elem instructions
slug: WebAssembly/Reference/Elem
page-type: landing-page
browser-compat: webassembly.elem
sidebar: webassemblysidebar
---

This set of pages details the instructions available in Wasm related to [elem](/en-US/docs/WebAssembly/Reference/Definitions/elem) definitions.

## Instructions

- [`elem.drop`](/en-US/docs/WebAssembly/Reference/Elem/drop)

>   - : Discards the data contained by a [passive](/en-US/docs/WebAssembly/Reference/Definitions/elem#passive_form) `elem` segment, freeing up its memory, after the associated `table` has been initialized via [`table.init`](/en-US/docs/WebAssembly/Reference/Table/init).

Reviewer 2: ```suggestion
  - : Discards the data contained by a [passive](/en-US/docs/WebAssembly/Reference/Definitions/elem#passive_form) `elem` segment, freeing up its memory, after being used in a [`table.init`](/en-US/docs/WebAssembly/Reference/Table/init).
```

Author: done
